DESCRIPTION

```diff
@@ -82,7 +84,7 @@ VignetteBuilder:
 Config/Needs/verdepcheck: haleyjeppson/ggmosaic, tidyverse/ggplot2,
     rstudio/shiny, insightsengineering/teal,
     insightsengineering/teal.transform, mllg/checkmate, tidyverse/dplyr,
-    rstudio/DT, tidyverse/forcats, r-lib/scales, daattali/shinyjs,
+    rstudio/DT, tidyverse/forcats, r-lib/rlang, r-lib/scales, daattali/shinyjs,
```

Comment:
As `rlang` was introduced as a dependency, it also needs to be added here and in `.pre-commit-config.yaml`